workshops/test/base.py

```diff
@@ -298,6 +298,17 @@ def _setUpEvents(self):
                              host=test_host,
                              admin_fee=100)
 
+        # create a full-blown event that got cancelled
+        e = Event.objects.create(start=event_start, end=event_end,
+                                 slug='starts-today-cancelled',
+                                 url='http://example.org/cancelled-event',
+                                 latitude=-10.0, longitude=10.0,
+                                 country='US', venue='University',
+                                 address='Phenomenal Street',
+                                 host=test_host)
+        tags = Tag.objects.filter(name__in=['SWC', 'cancelled'])
+        e.tags.set(tags)
```

Comment:
Depends what you understand by "can see". See a listing on all-events page? We don't test that kind of thing, because we assume [django-filter](http://django-filter.readthedocs.io/en/latest/) works correctly (and with these simple filters you actually only need to manually check if they work).

If you meant something like a queryset method (`Event.object.cancelled_events`) then there's no such method and therefore no corresponding test.

If you meant whether we can see cancelled events among unpublished events, then a test for that exists in `workshops.test.test_event.TestEvent.test_unpublished_events`.